PhotoFeed/app/src/main/java/com/mobileproto/jovanduy/photofeed/DbAccessor.java

```diff
@@ -0,0 +1,98 @@
+package com.mobileproto.jovanduy.photofeed;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.util.Log;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+/**
+ * Class to modify the database
+ */
+public class DbAccessor {
+
+    public FeedReaderDbHelper dbHelper;
+
+    public DbAccessor(Context context) {
+        this.dbHelper = new FeedReaderDbHelper(context);
+    }
+
+    /**
+     * add a url to the database
+     * @param url to be added to database
+     */
+    public void writeToDb(String url) {
+        SQLiteDatabase db = dbHelper.getWritableDatabase();
+        ContentValues values = new ContentValues();
+        values.put(FeedReaderContract.FeedEntry.COLUMN_NAME_URL, url);
+        try {
+            db.insertOrThrow(FeedReaderContract.FeedEntry.TABLE_NAME, null, values);
+        } catch (Exception ex) {
+            Log.e("Database Service", "Error occurred inserting into db", ex);
```

Comment:
Since you're doing this try catch maybe you could make this method return a boolean and return false if there is an exception. That way the rest of your code can know that you did not successfully insert into the DB and you can handle that.